### PR TITLE
Cleanup wall messages emitted by logind and systemctl

### PR DESCRIPTION
**Pull request by keszybz**
Cleanup patches I wrote when working on #23574.

### CODE REVIEW

File: src/shared/pager.c [hunk 1]
```diff
@@ -219,31 +219,30 @@ void pager_open(PagerFlags flags) {
...
-                        _exit(EXIT_FAILURE);
+                        if (i < ELEMENTSOF(pagers) - 1) {
+                                execlp(pagers[i], pagers[i], NULL);
+                                log_full_errno(errno == ENOENT ? LOG_DEBUG : LOG_WARNING, errno,
+                                               "Failed to execute '%s', will try '%s' next: %m", pagers[i], pagers[i+1]);
+                        } else {
+                                /* Close pipe to signal the parent to start sending data */
+                                safe_close_pair(exe_name_pipe);
+                                pager_fallback();
+                                /* not reached */
```
Comment (yuwata) [inline src/shared/pager.c]: Maybe, better to add `assert_not_reached()`?

**Review by yuwata — APPROVED** [review]
LGTM.

File: src/login/logind-utmp.c [hunk 2]
```diff
@@ -70,11 +70,11 @@ static int warn_wall(Manager *m, usec_t n) {
 
         left = m->scheduled_shutdown_timeout > n;
 
-        r = asprintf(&l, "%s%sThe system is going down for %s %s%s!",
+        r = asprintf(&l, "%s%sThe system will %s %s%s!",
```
Comment (yuwata) [inline src/login/logind-utmp.c:70]: Please update test-shutdown.py, which expects the translated messages, e.g. https://github.com/systemd/systemd/blob/1c1cb8ec6f16c9c38806b2aba86e4d80e569b0d7/test/test-shutdown.py#L57

**Comment by keszybz** [comment]
Should be fixed now.

**Pull request merged**